dev-python/pydispatcher/pydispatcher-2.0.1.recipe

```diff
@@ -1,25 +1,63 @@
-DESCRIPTION="
-Multi-producer-multi-consumer signal dispatching mechanism in python.
-"
+SUMMARY="Multi-producer-multi-consumer signal dispatching mechanism in Python"
+DESCRIPTION="PyDispatcher provides the Python programmer with a \
+multiple-producer-multiple-consumer, signal-registration and routing infrastructure for \
+use in multiple contexts. The mechanism of PyDispatcher started life as a highly rated \
+recipe in the Python Cookbook. The SourceForge project aims to include various \
+enhancements to the recipe developed during use in various applications."
 HOMEPAGE="http://pydispatcher.sourceforge.net/"
+COPYRIGHT="2001-2006, Patrick K. O'Brien and Contributors"
+LICENSE="BSD (3-clause)"
+REVISION="2"
 SOURCE_URI="http://downloads.sourceforge.net/project/pydispatcher/pydispatcher/2.0.1/PyDispatcher-2.0.1.tar.gz"
```

Comment:
This is not strictly required, but since we're updating this recipe it would be nice to apply this change:
```diff
-SOURCE_URI="http://downloads.sourceforge.net/project/pydispatcher/pydispatcher/2.0.1/PyDispatcher-2.0.1.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/pydispatcher/PyDispatcher-$portVersion.tar.gz"
```